packages/extensions/package.json

```diff
@@ -77,15 +79,18 @@
     "prosemirror-highlight": "^0.8.0",
     "prosemirror-search": "^1.0.0",
     "prosemirror-tables": "^1.4.0",
-    "shiki": "^1.12.0"
+    "shiki": "^1.12.0",
+    "y-prosemirror": "^1.2.9"
```

Comment:
Let's move `y-prosemirror` to `peerDependncies` and also add it in `peerDependenciesMeta`.